pkg/jobmanager/options.go

```diff
@@ -0,0 +1,36 @@
+package jobmanager
+
+import (
+	"github.com/facebookincubator/contest/pkg/api"
+)
+
+// Option is an additional argument to method New to change the behavior
+// of the JobManager.
+type Option interface {
+	a(*config)
+}
+
+type config struct {
+	apiOptions []api.Option
+}
+
+// OptionEventTimeout defines time duration for API request to be processed.
+// If a request is not processed in time, then an error is returned to the
+// client.
+type OptionAPI struct {
+	api.Option
+}
+
+// Apply implements Option.
```

Comment:
Function name and comment are not matching

Reply:
Oh, I pushed a corrupted code. It should've been `apply` (not `a`). Fixed. Thank you!